### PR TITLE
Article plugin styling

### PR DESCRIPTION
**Pull request by Dietr**
- Add annotations for titles
- Add layout for the title and parts

### CODE REVIEW

**Comment by nvdk** [comment]
@Dietr note that I requested oscar to change some of the classes, so likely this will need to be updated as well.

File: app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss [hunk 1]
```diff
@@ -600,7 +624,7 @@
   }
 
   [typeof='besluit:Artikel'] [property='prov:value']:before {
```
Comment (lagartoverde) [inline app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss:626]: besluit:Artikel is now say:Article and prov:value should be say:body

File: app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss [hunk 2]
```diff
@@ -189,6 +196,26 @@
...
+  [resource="http://mu.semte.ch/vocabularies/ext/Title"] + [property="say:heading"]:before {
+    content: 'Titel';
+  }
+
+  [resource="http://mu.semte.ch/vocabularies/ext/Chapter"] + [property="say:heading"]:before {
+    content: 'Hoofdstuk';
+  }
+
+  [resource="http://mu.semte.ch/vocabularies/ext/Section"] + [property="say:heading"]:before {
+    content: 'Afdeling';
```
Comment (abeforgit) [inline app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss:212]: Isn't this an odd translation of "section"?

**Review by abeforgit — CHANGES_REQUESTED** [review]
Tried to test with npm linking, it didnt work (the lines on the left showed up but without content) 
However it could be due to my setup, not sure.

File: app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss (hunk 2 above)
Comment (Dietr) [reply app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss:212]: That's the legal name for it that I have to follow.

**Pull request merged**